src/app/Layout/Presentation/Views/DistrictOverview.tsx

```diff
@@ -1,50 +1,105 @@
 ﻿import * as React from "react";
 import ReactTable from "react-table";
 import { DistrictResult } from "../../Interfaces/Results";
+import { toSum, toMean, toMax, toMin } from "../Utilities/ReduceUtilities";
 
 export interface DistrictOverviewProps {
     districtResults: DistrictResult[];
+    districtWidth: number;
 }
 
 export class DistrictOverview extends React.Component<DistrictOverviewProps, {}> {
     render() {
+        const data = this.props.districtResults;
+        const highestVotingPower = data.map((value) => value.votesPerSeat).reduce(toMin);
+        const lowestVotingPower = data.map((value) => value.votesPerSeat).reduce(toMax);
+        const averageVotingPower = data.map((value) => value.votesPerSeat).reduce(toMean);
         return (
-            <ReactTable
-                className="-highlight -striped"
-                defaultPageSize={19}
-                showPaginationBottom={false}
-                data={this.props.districtResults}
-                columns={[
-                    {
-                        Header: "Fylke",
-                        accessor: "name"
-                    },
-                    {
-                        Header: "Stemmer",
-                        accessor: "votes"
-                    },
-                    {
-                        Header: "Prosent",
-                        accessor: "percentVotes"
-                    },
-                    {
-                        Header: "Distrikt",
-                        accessor: "districtSeats"
-                    },
-                    {
-                        Header: "Utjevning",
-                        accessor: "levelingSeats"
-                    },
-                    {
-                        Header: "Sum",
-                        accessor: "totalSeats"
-                    },
-                    {
-                        Header: "Stemmer/mandat",
-                        accessor: "votesPerSeat"
-                    }
-                ]}
-            />
+            <React.Fragment>
+                <h2>Fylkesoversikt</h2>
+                <span>
+                    {"En stemme i "}
+                    <strong>{data.find((entry) => entry.votesPerSeat === highestVotingPower)!.name}</strong>
+                    {" hadde mest innflytelse, og telte "}
+                    {((1 / highestVotingPower / (1 / averageVotingPower)) * 100).toFixed(2) + "%"}
```

Comment:
I would suggest doing these calculations outside the return, to make it more readable.
A rule of thumb could be that, if you can intuitively see what the function does with a quick look, it may be in the return, but if it requires you to go through it more slowly you should split it out.
Also maybe take into account the selected number of decimals, rather than hard coding it to 2.
The same goes for the similar cases below.

```ts
const decimals = this.props.decimals;
const mostInfluentialCount = ((1 / highestVotingPower / (1 / averageVotingPower)) * 100).toFixed(decimals) + "%";
...
{" hadde mest innflytelse, og telte "}
{mostInfluentialCount}

```